### PR TITLE
Fix things to work with Jekyll templating

### PR DESCRIPTION
**Pull request by marumari**
Let's have another go at it!

### CODE REVIEW

File: common/common.conf [hunk 1]
```diff
@@ -5,7 +5,7 @@
 add_header "Cache-Control" "no-store";
 
 location /test {
-  root /var/www/badssl/common;
+  root /var/www/badssl/_site/common;
```
Comment (Hoverbear) [inline common/common.conf]: Was having this issue too.

**Comment by lgarron** [comment]
I'd like to merge this, but I'm having trouble combining it with my deployment. Give me a short while to reconcile?

**Comment by marumari** [comment]
No problem.  I *think* if you just do jekyll build and upload that _site directory that everything should work fine.  You'll have two copies of things up there, but the overall space consumption isn't much.

That said, I don't know how your actual implementation is done on the production system.  :)

**Comment by marumari** [comment]
If we could also configure jekyll to only build .html files, then we could save even more space!

**Comment by lgarron** [comment]
> I think if you just do jekyll build and upload that _site directory that everything should work fine.

The problem is more that the nginx config files reference both `badssl/_site` and just `badssl`.

**Comment by marumari** [comment]
Yeah, but mine works that way.  The files are duplicated in both places (with obviously the templated html files in _site), so it all should work.

**Comment by lgarron** [comment]
Merged with some rebasing.

Part of the problem was my old friend, the fact that `rsync --delete` that doesn't deleted excluded paths.

**Pull request closed without merging**